cost/superseded_instance_remediation/superseded_instance_remediation.pt

```diff
@@ -181,19 +177,27 @@ end
 policy "policy_superseded_instance" do
   validate $ds_combined_instance_data do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Instances will be resized from {{ parameters.param_old_instance_type }} to {{ parameters.param_new_instance_type }}"
-    detail_template <<-EOS
-# Superseded Instances
-| Resource UID | Name | Cloud | Old Instance Type | New Instance Type |
-| ------------ | ---- | ----- | ----------------- | ----------------- |
-{{ range data -}}
-| {{ .resource_uid }} | {{ .name }} | {{ .cloud_name }} | {{ parameters.param_old_instance_type }} | {{ parameters.param_new_instance_type }} |
-{{ end -}}
-___
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
     check eq(size(data), 0)
-	escalate $email
+	  escalate $report_superseded_instance_email
     escalate $esc_resize_instances
+    export do
+      resource_level true
+      field "id" do
```

Comment:
Id should be the last column.  datacenter should be added as the second column.  cloud should be the first column.  should be sorted by cloud, then datacenter.

Reply:
Changed the ID to last column, cloud as first column. The policy does not get the datacenter field. Id that field that should be added to the policy. I not not sure how to sort the export. Can you point to some example policy which has done this on a the export?